sdk/python/kubeflow/training/utils/utils.py

```diff
@@ -195,6 +195,9 @@ def get_container_spec(
     args: Optional[List[str]] = None,
     resources: Union[dict, models.V1ResourceRequirements, None] = None,
     volume_mounts: Optional[List[models.V1VolumeMount]] = None,
+    env: Optional[
```

Comment:
Let's use the env_vars here as well to be consistent.
```suggestion
    env_vars: Optional[
```